test/js/specs/happy.js

```diff
@@ -207,6 +207,15 @@ describe('Happy Paths', options, ({driver, pageObjects}) => {
       uploadFileAndClickConfirmButton('face.jpeg')
       verificationComplete.verifyUIElements(verificationCompleteCopy)
     })
+
+    it('should be able to submit a document without seeing the document selector screen', async () => {
+      driver.get(localhostUrl + `?oneDoc=true&?async=false&useWebcam=false`)
```

Comment:
You don't need the question mark after `oneDoc=true`. It should be `?oneDoc=true&async=false&useWebcam=false`

Reply:
Was it picking up the following queries?

Reply:
Good spot. 
Yeah, it was picking up the next queries as I'm uploading selfie jpeg instead of capture.
Will change that anyway and remove `?`